django_apscheduler/admin.py

```diff
@@ -39,6 +46,20 @@ def average_duration(self, obj):
 
     average_duration.short_description = "Average Duration (sec)"
 
+    actions = ['run_selected_jobs']
+
+    def run_selected_jobs(self, request, queryset):
+        scheduler = BackgroundScheduler()
+        scheduler.add_jobstore(DjangoJobStore(), "default")
```

Comment:
Using a custom [`MemoryJobStore`](https://apscheduler.readthedocs.io/en/stable/modules/jobstores/memory.html#apscheduler.jobstores.memory.MemoryJobStore) here should solve the problem of schedulers stepping on each other.

It may even be possible to add the `DjangoJobExecution` mixin to the memory job store so that the the `DjangoJobExecution` entries are still generated for logging purposes.